app/Http/Requests/ResourceItem/StoreResourceItemRequest.php

```
@@ -37,15 +47,24 @@
+            'quantity' => 'required|regex:/^\d+(\.\d{1,2})?$/|numeric',
             'unit' => 'required|string',
-            'unit_cost' => 'required|regex:/^\d+(\.\d{1,2})?$/',
+            'unit_cost' => 'required|regex:/^\d+(\.\d{1,2})?$/|numeric',
             'resource_count' => 'required|integer',
-            'consumption_rate' => 'nullable|regex:/^\d+(\.\d{1,2})?$/',
+            'consumption_rate' => 'nullable|regex:/^\d+(\.\d{1,2})?$/|numeric',
             'consumption_unit' => 'nullable|string',
             'labor_cost_category' =>  ['nullable', new Enum(LaborCostCategory::class)],
             'work_time_category' => ['nullable', new Enum(WorkTimeCategory::class)],
             'remarks' => 'nullable|string',
+            'status' => ['nullable', Rule::in(ResourceStatus::toArray())]
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (undefined)

(StaticAccess)